api/src/main/java/com/codeforcommunity/rest/FailureHandler.java

```diff
@@ -1,17 +1,6 @@
 package com.codeforcommunity.rest;
 
-import com.codeforcommunity.exceptions.CreateUserException;
-import com.codeforcommunity.exceptions.EmailAlreadyInUseException;
-import com.codeforcommunity.exceptions.ExpiredSecretKeyException;
-import com.codeforcommunity.exceptions.HandledException;
-import com.codeforcommunity.exceptions.InvalidSecretKeyException;
-import com.codeforcommunity.exceptions.MalformedParameterException;
-import com.codeforcommunity.exceptions.MissingHeaderException;
-import com.codeforcommunity.exceptions.MissingParameterException;
-import com.codeforcommunity.exceptions.TokenInvalidException;
-import com.codeforcommunity.exceptions.UsedSecretKeyException;
-import com.codeforcommunity.exceptions.UserDoesNotExistException;
-import com.codeforcommunity.exceptions.UsernameAlreadyInUseException;
+import com.codeforcommunity.exceptions.*;
```

Comment:
This isn't a huge deal but I'd suggest never using wildcard imports. You can change the setting on Intellij so that it doesn't do this automatically.
Wildcards aren't great because it can make it harder to tell where certain code is coming from. And because 90% of the time imports are going to be collapsed it doesn't really matter how long our import list is.

Reply:
Yeah that was done automatically, I'll fix it in a separate PR, I broke this one